beacon-chain/rpc/prysm/validator/types/attestation_data.go

```diff
@@ -0,0 +1,33 @@
+package types
```

Comment:
`rpc` is the outer-layer of the codebase, and as such types defined here should not be used in other packages